op-deployer/book/src/user-guide/verify.md

```diff
@@ -1,17 +1,42 @@
 # The Verify Command
 
-Once you have deployed contracts via [bootstrap][bootstrap], you can use the `verify` command to verify the source code on Etherscan. Constructor args used in the verification request are extracted automatically from contract initcode via the tx that created the contract.
+Once you have deployed contracts via [bootstrap][bootstrap] or [apply][apply], you can use the `verify` command to verify the source code on block explorers like Etherscan or Blockscout. The command uses the `forge verify-contract` binary, which automatically handles constructor argument detection and source code verification.
 
 [bootstrap]: bootstrap.md
+[apply]: apply.md
 
 You can call the `verify` command like this:
 
 ```shell
 op-deployer verify \
   --l1-rpc-url <l1 rpc url> \
-  --input-file <filepath to input .json file> \
-  --etherscan-api-key <your free etherscan api key> \
-  --artifacts-locator <l1 forge-artifacts locator>
+  --input-file <filepath to input .json or state.json file> \
+  --etherscan-api-key <your api key> \
+  --artifacts-locator <l1 forge-artifacts locator> \
+  --verifier etherscan
+```
+
+For Blockscout verification (uses default URLs for mainnet/sepolia):
+
+```shell
+op-deployer verify \
+  --l1-rpc-url <l1 rpc url> \
+  --input-file <filepath to input .json or state.json file> \
+  --etherscan-api-key <your api key> \
+  --artifacts-locator <l1 forge-artifacts locator> \
+  --verifier blockscout
+```
+
+For custom block explorer verification:
+
+```shell
+op-deployer verify \
+  --l1-rpc-url <l1 rpc url> \
+  --input-file <filepath to input .json or state.json file> \
+  --etherscan-api-key <your api key> \
+  --artifacts-locator <l1 forge-artifacts locator> \
+  --verifier custom \
+  --verifier-url <custom blockscout api url>
```

Comment:
Does this have to be a blockscout url, or can it be any block explorer that supports etherscan api/v2?

Reply:
That was a typo, should be any block explorer that supports etherscan v2